keel-web/src/main/kotlin/com/netflix/spinnaker/keel/apidocs/_apidocs.kt

```diff
@@ -0,0 +1,14 @@
+package com.netflix.spinnaker.keel.apidocs
+
+import com.fasterxml.jackson.databind.type.TypeBase
+import io.swagger.v3.core.converter.AnnotatedType
+import java.lang.reflect.ParameterizedType
+
+internal val AnnotatedType.baseType: Class<*>
```

Comment:
Neat! I didn't know you could do this and have the extension be imported automatically in all files of the package. Is that a naming convention with `_` in front of the package name, or does anything with an underscore get imported?

Reply:
No, it’s just because that function is a top level member of the package. I always struggle with what to name files that just contain functions. I thought this was as good a way of any as distinguishing it from a file containing a class.